src/android/InAppBrowser.java

```diff
@@ -797,6 +872,49 @@ private void sendUpdate(JSONObject obj, boolean keepCallback, PluginResult.Statu
         public InAppBrowserClient(CordovaWebView webView, EditText mEditText) {
             this.webView = webView;
             this.edittext = mEditText;
+            this.currentUrl = null;
+            this.whiteList = new ArrayList<String>();
+        }
+
+        /**
+         * Ignore SSL Certificate errors when the certificate is good with HttpsUrlConnection
+         */
+        @Override
+        public void onReceivedSslError(WebView view, SslErrorHandler handler, SslError error) {
+            Log.d(LOG_TAG, "We have received an SSL error on this URL: " + error.getUrl());
+            int endIndex = error.getUrl().indexOf("/", 8);
```

Comment:
@brad I think `8` is from the length of `https://`? What do you think about refactoring here to make that more clear?